kusama-guide/sidebars.js

```diff
@@ -172,8 +172,13 @@ module.exports = {
                 "general/transaction-attacks",
               ],
             },
+            {
+              type: 'link',
+              label: 'Research Pages',
```

Comment:
maybe we can change the label to "W3F Research" or "Web3 Foundation Research"?

```suggestion
              label: 'Web3 Foundation Research',
```